website_docs/getting-started.md

```diff
@@ -184,16 +184,15 @@ Start by instrumenting the `Run` method.
 // Run starts polling users for Fibonacci number requests and writes results.
 func (a *App) Run(ctx context.Context) error {
 	for {
-		var span trace.Span
-		ctx, span = otel.Tracer(name).Start(ctx, "Run")
+		newCtx, span := otel.Tracer(name).Start(ctx, "Run")
```

Comment:
This makes sense. But it may make less sense to someone discovering the library. How about adding a comment explaining the reason for that new variable?